spec/system/candidate_interface/entering_details/candidate_changing_their_gcse_type_from_completed_to_missing_spec.rb

```diff
@@ -4,9 +4,7 @@
   include CandidateHelper
 
   scenario 'Candidate completes their maths GCSE and then changes the type to missing and then back to a GCSE' do
-    given_i_am_signed_in
-
-    when_i_visit_the_candidate_application_page
```

Comment:
Are we using this method anywhere in this test now? If not, let's delete it from the file so we don't have unused code.